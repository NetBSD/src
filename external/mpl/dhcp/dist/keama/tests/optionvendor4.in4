# group and class declaration config

# vendor option space
option space foobar;

option vendor.foobar code 12345 = encapsulate foobar;
option foobar.test code 1 = text;
option foobar.test "a test";
