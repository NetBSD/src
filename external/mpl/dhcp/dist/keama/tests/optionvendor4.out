{
  # group and class declaration config
  # vendor option space
  "Dhcp4": {
    "option-def": [
      {
        "space": "vendor-12345",
        "name": "test",
        "code": 1,
        "type": "string"
      }
    ],
    "option-data": [
      {
        "space": "dhcp4",
        "name": "vivso-suboptions",
        "code": 125,
        "data": "12345"
      },
      {
        "space": "vendor-12345",
        "name": "test",
        "code": 1,
        "data": "a test"
      }
    ]
  }
}
